src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1546,9 +1556,11 @@ namespace winrt::TerminalApp::implementation
     {
         if (auto index{ _GetFocusedTabIndex() })
         {
-            auto focusedTab{ _GetStrongTabImpl(*index) };
-            _UnZoomIfNeeded();
-            focusedTab->ResizePane(direction);
+            if (auto terminalTab = _GetTerminalTabImpl(_tabs.GetAt(*index)))
```

Comment:
There's a lot of this `if (get focused tab index) { if (get tab from index) { tab.doSomething() } }` going on in here. Would it be better to have a helper that just does it all in one go?

Reply:
I’d rather not add that now to reduce the merge burden when this flows _back_ into SUI (since right now i can use the “take all `ours`” strategy), but it is worth doing.